app/common/public/locales/en/translation.json

```diff
@@ -281,5 +281,13 @@
   "invalidCapType": "Invalid capability type: {{type}}",
   "whitespaceDetected": "Text Starts and/or Ends With Whitespace",
   "pcloudyCredentialsRequired": "pCloudy username and api key are required!",
-  "duplicateCapabilityNameError": "A capability set with the same name already exists"
+  "duplicateCapabilityNameError": "A capability set with the same name already exists",
+  "errorLoadingGestures": "Error loading gestures",
+  "unableToUploadGestureFiles": "Could not upload gesture from below files:",
+  "gestureEmptyTickError": "Actions should have valid ticks.",
+  "gestureInvalidEventError": "Invalid event type '{{invalidEvent}}'. Valid values are {{validEvents}}.",
+  "gestureRequiredFieldsError": "Value '{{fields}}' are required for '{{eventType}}' event.",
```

Comment:
'Value' should be in plural

Reply:
Consider using the native plurals support. The reason for it is that other translations may have different rules for plurals creations. Check https://www.i18next.com/translation-function/plurals for more details.

Reply:
I think the i18next plurals functionality would add a lot of complexity to this PR, so I would suggest leaving that for a separate PR.